requirements-dev.txt

```diff
@@ -1,2 +1,3 @@
-pytest==2.7.2
+pytest==2.9.2
 pytest-cov==2.1.0
+git+git://github.com/tilery/utilery.git
```

Comment:
I'm going to pull this and then I'll update test_utils to reference aiovectortiler.